src/Backend/Modules/Location/Actions/Index.php

```diff
@@ -144,6 +155,12 @@ protected function loadSettingsForm()
         $this->form->addText('width', $this->settings['width']);
         $this->form->addText('height', $this->settings['height']);
         $this->form->addDropdown('map_type', $mapTypes, $this->settings['map_type']);
+        $this->form->addDropdown(
+            'map_style',
+            $mapStyles,
+            (isset($this->settings['map_style']))
```

Comment:
ou don't need the extra brackets and it can be added on the same line like
```
$this->form->addDropdown(
    'map_style',
    $mapStyles,
    isset($this->settings['map_style']) ? $this->settings['map_style'] : null
)
```